src/NetAnalyzers/UnitTests/Microsoft.NetCore.Analyzers/Performance/UsePropertyInsteadOfCountMethodWhenAvailableTests.cs

```diff
@@ -646,5 +647,37 @@ public class SomeClass
                     .WithLocation(8, 23)
                     .WithArguments(nameof(IReadOnlyCollection<int>.Count)));
         }
+
+        [Fact, WorkItem(3724, "https://github.com/dotnet/roslyn-analyzers/issues/3724")]
+        public static async Task PropertyAccessParentIsNotAlwaysDirectlyTheInvocation()
+        {
+            await VerifyCS.VerifyAnalyzerAsync(@"
+using System.Linq;
+using System.Text;
+
+public class C
+{
+    public static bool IsChildPath(string parentPath, string childPath)
+    {
+        return (IsDirectorySeparator(childPath[parentPath.Length]) || IsDirectorySeparator(childPath[{|CA1829:parentPath.Count()|}]));
+    }
+
+    public static bool IsDirectorySeparator(char c) => false;
+}");
+
+            await VerifyVB.VerifyAnalyzerAsync(@"
+Imports System.Linq
+
+Public Class C
+    Public Shared Function IsChildPath(ByVal parentPath As String, ByVal childPath As String) As Boolean
+        Return (IsDirectorySeparator(childPath(parentPath.Length)) OrElse IsDirectorySeparator(childPath(parentPath.Count())))
```

Comment:
```suggestion
        Return (IsDirectorySeparator(childPath(parentPath.Length)) OrElse IsDirectorySeparator(childPath({|CA1829:parentPath.Count()|})))
```

This seems to be the reason of CI failure.

Reply:
Got it! I added the vbnet code without running the tests to ensure everything was alright 🤦